paket.dependencies

```diff
@@ -1,10 +1,12 @@
 version 7.2.1
+storage: none
 source https://api.nuget.org/v3/index.json
 frameworks: net6.0, netstandard2.0
 
 nuget FSharp.Core ~> 6 // We need task{} CE from F# 6.0
-nuget System.Text.Json ~> 6
 nuget FSharp.SystemTextJson
+nuget FSharp.TypeProviders.SDK >= 8
```

Comment:
1. It is already a recommended way to use SDK? Dependencies always were a pain to TPs
2. What is your motivation for this change?